tests/serialized_test_data_generation/submit_job.sh

```diff
@@ -56,10 +59,18 @@ cp $RUNDIR/logfile.*.out $TEST_DATA_DIR/
 cp $RUNDIR/std*.out $TEST_DATA_DIR
 env > $TEST_DATA_DIR/env.out
 
+# copy data to mounted dir
+if [ "$TEST_DATA_DIR" != "$MOUNTED_DATA_DIR" ] ; then
+  mv $TEST_DATA_DIR/* $MOUNTED_DATA_DIR/
+fi
```

Comment:
The main reason why serialization is so slow, is because we are continually opening/closing files (mostly because of not doing internal buffering for the j-blocked cases). So moving the serialization into a tmpfs gives a huge boost (temporary disk in memory).

The original version of the script assumes that $TEST_DATA_DIR exists. This is only executed in case $TEST_DATA_DIR is not equal to $MOUNTED_DATA_DIR, so I think we are ok. Or am I missing something?

Reply:
Well both are hard coded in the script, so it will always be true that they aren't equal. That's fine. I didn't notice the tmpfs part in the first review, but you did mention it in standup. Ok great! Glad this makes it faster.